src/i18n/locales/en/translation.json

```
@@ -637,7 +637,12 @@
       "title": "Nostr",
       "allow": "Allow {{host}} to:",
       "read_public_key": "Read your public key",
-      "block_and_ignore": "Block and ignore {{host}}"
+      "block_and_ignore": "Block and ignore {{host}}",
+      "confirm_sign_message": {
+        "remember": {
+          "label": "Remember this decission"
```

```suggestion
          "label": "Remember this decision"
```

or maybe "Remember and don't ask again"?